doc/factory/configuration.html

```diff
@@ -964,6 +967,55 @@ <h3 class="western"><a name="args_global"></a>Global arguments</h3>
               <a href="custom_scripts.html">custom scripts page</a>.
             </p>
           </li>
+          <li>
+            <a name="cvmfsexec_distro" />
+            <div class="xml">
+              &lt;glidein&gt;&lt;cvmfsexec_distro
+              sources=&quot;<i>sources_list</i>&quot;
+              platforms=&quot;<i>platforms_list</i>&quot; /&gt;
+            </div>
+
+            <p>
+              This knob provides control over the building and/or rebuilding of
+              cvmfsexec distributions when CVMFS is to be provisioned on demand.
+              Distributions built via this knob will be located in factory work
+              directory location (by default, for an RPM-based install of
+              GlideinWMS, this would typically be
+              <i>/var/lib/gwms-factory/work-dir</i>). Two attributes are
```

Comment:
Somewhere say the name of the subdirectory with the distributions

Reply:
updated to include this detail